CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ The changelog for `IGListKit`. Also see the [releases](https://github.com/instag
 
 - Added `IGListCollectionScrollingTraits` for exposing `UICollectionView` scrolling traits to section controllers via `IGListCollectionContext`. [Adam Stern](https://github.com/adamastern) (tbd)
 
+- Support reloading a `IGListBindingSectionController`. [Jeff Bailey](https://github.com/jeffbailey) (tbd)
```

Comment:
2 nits:

- Replace tbd with the number and link to this PR (1213)
- Let's make the line a little more descriptive. Something like "no longer asserts when reloading..." etc etc.